plankton/core/utils.py

```diff
@@ -202,3 +203,20 @@ def failing_init(obj, *args, **kwargs):
                             for name in names)
 
         return objects if len(objects) != 1 else objects[0]
+
+
+def format_doc_text(text):
+    """
+    A very thin wrapper around textwrap.fill to consistently wrap documentation text
+    for display in a command line environment. The text is wrapped to 99 characters with an
+    indentation depth of 4 spaces. Each line is wrapped independently in order to preserve
+    manually added line breaks.
+
+    :param text: The text to format
+    :return: The formatted doc text
+    """
+
+    return '\n'.join(
+        textwrap.fill(line, width=99, initial_indent='    ', subsequent_indent='    ')
+        for line in
+        text.splitlines())
```

Comment:
I'm not too happy with `textwrap.fill`, it seems to be a problem wrapping text that already has linebreaks, which is the only reason for this splitting and re-combinding.

All the docstrings are obtained by `inspect.getdoc`. According to the documentation (if I remember correctly) that does apply `cleandoc` to the docstring. It could be added to the documentation that the function expects a "clean" docstring.